src/ifcparse/ArgumentType.h

```diff
@@ -17,44 +17,43 @@
 *                                                                              *
 ********************************************************************************/
 
-#include "../ifcparse/IfcSchema.h"
-
 #include "ifc_parse_api.h"
+#include "IfcSchema.h"
```

Comment:
These need to go after the include guard.

Reply:
agreed. not sure what happened here, maybe some merge artefact, or just a weird oversight.